index.d.ts

```diff
@@ -1488,7 +1490,7 @@ declare namespace Eris {
   }
 
   export class Member extends Base {
-    activities: Activity[];
+    activities?: Activity[];
     clientStatus: ClientStatus;
```

Comment:
I believe this will also be optional??
```suggestion
    clientStatus?: ClientStatus;
```